OfficeDev/office-ui-fabric-react
# fix(Toolbar) :  Toolbar menu item doesn't focus menu trigger element after 'onClick' was executed

#### Pull request checklist

- [x ] Addresses an existing issue: Fixes https://github.com/microsoft/fluent-ui-react/issues/2249

#### Description of changes
Before we moved focus back on the trigger menu button when we been closing the menu. 
Now we don't handle focus when closing the menu and consumer of toolbar has to handle them self. 

#### Focus areas to test
How to test?:
1. into onClick add logic for any element focus
2. execute click on give item in menu
Expected: focus moved to the element which you defined in "onClick" 


###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12266)
